docs/pulse/drill-down.md

```diff
@@ -0,0 +1,75 @@
+
+The exact calculation for each of the above depends on the metric type: 
+
+| Metric Type      | Total Calculation       | Mean           | Units          |
+|------------------|-------------------------|----------------|----------------|
+| event_count      | Sum of events (99.9% winsorization)           | Average events per user (99.9% winsorization)  | All users  |
+| event_dau        | Sum of event DAU (distinct user-day pairs) | Average event_dau value per user per day. Note that we call this "Event Participation Rate" as this can be interpreted as the probability a user is DAU for that event.  | All users |
+| ratio            | Overall ratio: sum(numerator values)/sum(denominator values)  | Overall ratio  |  Participating users |
+| sum              | Total sum of values (99.9% winsorization)     | Average value per user (99.9% winsorization)   | All users |
+| mean             | Overall mean value    | Overall mean value   | Participating users |
+| user: dau, wau, mau_28day |  Not shown   | Average metric value per user per day. The probability that a user is xAU | All users |
+| user: new_dau, new_wau, new_mau_28day |  Count of distinct users that are new xAU at some point in the experiment  | Fraction of users that are new xAU |  All users |
+| user: retention metrics |  Overall average retention rate   | Overall average retention rate  | Participating users |
+| user: L7, L14, L28 |  Not shown   | Average L-ness value per user per day  | All users |
+
+## Detailed View
+
+Click on **View Details** to access in depth metric information.  The deatiled view contains three sections: 
+- **Time Series**: How the metrics evolve over time
+- **Raw Date**: Group level statistics
+- **Impact**: How the experiment impacts the metric 
+
+### Time Series
+
+![image](https://user-images.githubusercontent.com/90343952/171748363-20d7819c-e264-4135-a126-93341e4bb717.png)
+
+In this view, select and drag as needed to zoom-in on different time ranges.  Three types of time series are avaialble in the drop-down:
+- Daily: The metric impact on each calendar day without aggregating days together.  This is useful for assessing the variability of the metric day-over-day and the impact of specific events.  It's the recommended time series view for Holdouts, since it highlights the impact over time as new features are launched. 
+- Cumulative: Shows how the aggregated metric lift and confidence intervals evolve over time.  
+- Days Since Exposure: Shows the metric impact broken down by the number of days a user has been in the experiment, which helps distinguish between novelty effects and sustained impact.  For key metrics, this view also shows pre-experiment metrics to identify an differences that existed between groups even before the experiment started.  
```

Comment:
"number of days a user has been in the experiment" -> but this is an aggregate for the metrics computed across all users... would help to slightly better tease out if it's "days since start of experiment exposures to any user", perhaps?